brage-import/src/test/resources/features/FileMerger.feature

```diff
@@ -63,7 +65,7 @@ Feature:
     Then the merged nva publication has a file with values:
       | filename | identifier                           | mimeType                                                                  | size | lisense | publisherVersion | embargoDate | publishedDate                  |
       | a.pptx   | 9ba7b547-36ee-467b-b145-12bc5b3b9d9e | application/vnd.openxmlformats-officedocument.presentationml.presentation | 123  | null    | null              | null        | 2023-10-20T11:48:59.131046060Z |
-    And the merged nva publication has a handle equal to "https://hdl.handle.net/11250/2506045"
+    And the merged nva publication has a handle equal to "https://hdl.handle.net/11250/2506045" in additional identifiers
```

Comment:
It's really important you read this test carefully and understand what this change effectively does.